tests/controls/dialogtest.cpp

```diff
@@ -36,11 +36,13 @@ class ModalDialogsTestCase : public CppUnit::TestCase
 #endif
         CPPUNIT_TEST( FileDialog );
         CPPUNIT_TEST( CustomDialog );
+        CPPUNIT_TEST( wxInitDialog );
```

Comment:
Minor comment: it's a bit strange to use `wx` prefix for a method, we never do this anywhere else.